app/controllers/registration/early_years_experiences_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Registration
+  class EarlyYearsExperiencesController < BaseController
+    def edit
+      track('registration_early_years_experience')
```

Comment:
Only track in `update` instead.

Reply:
[73b1a5b](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1048/commits/73b1a5b456f09f3b1cee1b4afe0e8287d4f7a418)